core/modules/wsched/ChunkDisk.h

```diff
@@ -63,16 +63,21 @@ class ChunkDisk {
     std::size_t getSize() const;
 
     void setResourceStarved(bool starved);
+    bool nextTaskDifferentChunkId();
 
-    /// Class that keeps the minimum chunk id at the front of the heap.
+    /// Class that keeps the minimum chunkId at the front of the heap
+    /// and within that chunkId, start with the slowest tables to scan.
     class MinHeap {
     public:
         // Using a greater than comparison function results in a minimum value heap.
         std::function<bool(wbase::Task::Ptr const&, wbase::Task::Ptr const&)> compareFunc =
             [](wbase::Task::Ptr const& x, wbase::Task::Ptr const& y) -> bool {
```

Comment:
Why is not this a regular (static) function? `std::function` has some overhead, I'm not sure it's worth paying that price here.